app/server/command-line.js

```diff
@@ -2,75 +2,102 @@ const fs = require('fs');
 const path = require('path');
 const glob = require('glob');
 const rimraf = require('rimraf');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
 const docLoader = require('../shared/doc-loader');
+const { stringify: csvStringify } = require('csv-stringify/dist/cjs/sync.cjs');
 
 function checkCommandLine() {
-  const ejectIndex = process.argv.indexOf('eject');
-  if (ejectIndex !== -1) {
-    return {
-      commandName: 'eject',
-      args: process.argv.slice(ejectIndex + 1)
-    };
-  }
-
-  return null;
+  const cliArgs = yargs(hideBin(process.argv))
+    .command({
+      command: 'eject',
+      describe: 'Export one or multiple AnaVis files to JSON or CSV',
+      builder: yargs => yargs
+        .option('directory', { alias: 'd', demandOption: false, type: 'string' })
+        .option('format', { alias: 'f', demandOption: true, type: 'string', default: 'json' }),
```

Comment:
It's a bit strange that the not-demanded option has no default, but the demanded one does. Does "format" need to be demanded in this case, since it has a fallback? 🤔